### DISCUSSION TITLE
Explicit question

### DISCUSSION THREAD

**Discussion by tyriekfv** [RandomNinjaAtk/docker-lidarr-extended#194] [Q&A] [answered]
If an album is already at max upgrade (for me that's .flac) and it's a clean version. Will the script continue to look to add an explicit version? I ask because I believe Lidarr removes the album from wanted/missing category and I would prefer the explicit version but I know the script looks at that section. Or am I going to need to delete all the albums and redownload with the explicit option turned on?

**Comment by RandomNinjaAtk** [comment, marked as answer]
It only looks for what lidarr tells it to look for, so it would not search for it in your case.
